www/docs/main/quickstart.mdx

```diff
@@ -70,56 +71,114 @@ export type AppRouter = typeof appRouter;
-
-`npm install @trpc/client@next @trpc/server@next`
-
-For generating a powerful set of React hooks for querying your tRPC API. Powered by [@tanstack/react-query](https://tanstack.com/query/v4/docs/adapters/react-query).
-
-`npm install @trpc/react-query@next @tanstack/react-query`
-
-For a set of utilies for integrating tRPC with [Next.js](https://nextjs.org/).
-
-`npm install @trpc/next@next`
+tRPC is broken up into several packages so you can install only what you need. Make sure to install the packages you want in the proper sections of your codebase.
+
+<table width="100%">
+  <thead>
+    <tr>
+      <th>Purpose</th>
+      <th>Location</th>
+      <th>Install command</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Implement endpoints and routers</td>
+      <td>Server</td>
+      <td>
+        <code>npm install @trpc/server</code>
+      </td>
+    </tr>
+    <tr>
+      <td>Call procedures on client</td>
+      <td>Client</td>
+      <td>
+        <code>npm install @trpc/client @trpc/server</code>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        React hooks powered by{' '}
+        <a
+          href="https://tanstack.com/query/v4/docs/adapters/react-query"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          @tanstack/react-query
+        </a>{' '}
+      </td>
+      <td>Client</td>
+      <td>
+        <code>npm install @trpc/react-query @tanstack/react-query</code>
+      </td>
+      </tr>
+      <tr>
+      <td>
+        Next.js integration utilities </td>
+      <td>Next.js</td>
+      <td>
+        <code>npm install @trpc/next</code>
+      </td>
+    </tr>
+
+  </tbody>
+</table>
 
 ### Installation Snippets
 
-<!-- remark-plugin-npm2yarn doesn't play nice with shiki so have to do these tabs manually -->
-
-**npm**
-
-```bash
-npm install @trpc/server@next @trpc/client@next @trpc/react-query@next @trpc/next@next @tanstack/react-query
-```
-
-**yarn**
-
-```bash
-yarn add @trpc/server@next @trpc/client@next @trpc/react-query@next @trpc/next@next @tanstack/react-query
-```
-
-**pnpm**
-
-```bash
-pnpm add @trpc/server@next @trpc/client@next @trpc/react-query@next @trpc/next@next @tanstack/react-query
-```
-
-## Defining a router
-
-Let's walk through the steps of building a typesafe API with tRPC. To start, this API will only contain two endpoints:
+Here are some install scripts to add tRPC to your project. These scripts include every `@trpc/*` package so feel free to remove what you don't need!
+
+<!--
+    Shiki doesn't play nice in <Tabs /> so I had to rebuild it myself.
+    The styles are all coming off of the shiki stylesheet other than
+    "manual-nord" which can be found in /src/css/custom.css.
+ -->
+
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+    <pre class="shiki nord manual-nord">
+      <div class="code-container">
+        <div class="line">
+          npm install @trpc/server@next @trpc/client@next @trpc/react-query@next
```

Comment:
I think the rc is tagged under `@latest` so we should probably remove the `@next`.

Reply:
Thanks!

Reply:
Updated everywhere